modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsSouth.scala

```diff
@@ -101,14 +140,22 @@ object TcsSouth {
 
   @Lenses
   final case class TcsSeqConfig[F[_]](
-                                       guideWithP1: Option[StandardGuideOptions.Value],
-                                       guideWithP2: Option[StandardGuideOptions.Value],
-                                       guideWithOI: Option[StandardGuideOptions.Value],
-                                       offsetA: Option[InstrumentOffset],
-                                       wavelA: Option[Wavelength],
-                                       lightPath: LightPath,
-                                       instrument: InstrumentSystem[F]
-                                     )
+    guideWithP1: Option[StandardGuideOptions.Value],
+    guideWithP2: Option[StandardGuideOptions.Value],
+    guideWithOI: Option[StandardGuideOptions.Value],
+    guideWithCWFS1: Option[StandardGuideOptions.Value],
+    guideWithCWFS2: Option[StandardGuideOptions.Value],
+    guideWithCWFS3: Option[StandardGuideOptions.Value],
+    guideWithODGW1: Option[StandardGuideOptions.Value],
+    guideWithODGW2: Option[StandardGuideOptions.Value],
+    guideWithODGW3: Option[StandardGuideOptions.Value],
+    guideWithODGW4: Option[StandardGuideOptions.Value],
```

Comment:
I'm not really suggesting anything here for this PR I guess, but this one leaves me unquiet. Long lists of parameters where everything is the same type seem problematic because it is easy to mess up the order. Here there are 10 parameters with the same type. Also, it seems odd that all the different OIs combine down to one thing, `guideWithOI`, but other individual guider types are listed.  If we add a new guider I guess this class grows another `Option[StandardGuideOptions.Value]` parameter.  We could at least wrap them at some point to give them a distinct type.

Reply:
Ok, I'll put a new task for this.